### DISCUSSION TITLE
Provider Google Calendar copy/paste

### DISCUSSION THREAD

**Discussion by Chicco67** [kewisch/gdata-provider#503] [Support] [answered]
hi, 

after the last update of TH (102.1.2 32bit windows7) the copy / paste function of an already existing event no longer works.
has anyone else encountered the same problem?

Bye
Fred

**Comment by xjonx1** [comment, marked as answer]
There was an issue with the add-on and is now fixed by the developer. Details here: https://github.com/kewisch/gdata-provider/discussions/519#discussioncomment-4147395
